doc/changes/changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changes
 
 * [6.0.0](changes_6.0.0.md)
+* [5.0.6](changes_5.0.6.md)
```

Comment:
I think we should name it 6.0.1 or 7.0.0. Customers will always pick the latest one.

Reply:
I think for customers it's not important what happened. They just want the newest version to work properly.